keep/providers/jira_provider/jira_provider.py

```diff
@@ -304,6 +304,61 @@ def __create_issue(
             return {"issue": response.json()}
         except Exception as e:
             raise ProviderException(f"Failed to create an issue: {e}")
+        
+    def __update_issue(
+        self,
+        ticket_id: str,
+        summary: str,
+        description: str = "",
+        labels: List[str] = None,
+        components: List[str] = None,
+        custom_fields: dict = None,
+        **kwargs: dict,
+    ):
```

Comment:
missing priority

Reply:
Couldn't find a way for priority to work on jira cloud. Works good in self hosted versions

Reply:
https://community.developer.atlassian.com/t/change-priority-of-issue-via-api/70025/5 there seems to be a way, can you please try this?